charts/cert-manager/test/default.yaml.out

```diff
@@ -1116,6 +1118,25 @@ spec:
       nodeSelector:
         kubernetes.io/os: linux
 ---
+# Source: cert-manager/charts/cert-manager/templates/crds.yaml
+#
+# START crd
+---
+# Source: cert-manager/charts/cert-manager/templates/crds.yaml
+# START crd
+---
+# Source: cert-manager/charts/cert-manager/templates/crds.yaml
+# START crd
+---
+# Source: cert-manager/charts/cert-manager/templates/crds.yaml
+# START crd
+---
+# Source: cert-manager/charts/cert-manager/templates/crds.yaml
+# START crd
+---
+# Source: cert-manager/charts/cert-manager/templates/crds.yaml
+# START crd
+---
```

Comment:
Where does this come from? is there some CRDs we should be installing?

Reply:
@wurbanski Sorry, I saw this comment after approving the PR. @xrstf please take a look.